noggin/security/ipa_admin.py

```diff
@@ -70,7 +79,7 @@ def wrapper(*args, **kwargs):
 
     def __getattr__(self, name):
         wrapped_methods = list(self.__WRAPPED_METHODS)
-        if self.__app.config['TESTING']:  # pragma: no cover
+        if current_app.config.get('TESTING', False):  # pragma: no cover
```

Comment:
Putting in `False` as the default is superfluous, as `None` (the "default default" :wink:) would work fine. As well, shouldn't block.